server/queries/loanQueries.js

```diff
@@ -36,5 +36,17 @@ const loans = {
       });
     }
   },
+  getRepaymentById(req, res) {
```

Comment:
I believe this method/function gets (by ID) a single repayment history

Reply:
yes, it does... do you want me to rename it?